frontend/locales/de.json

```diff
@@ -74,54 +40,88 @@
+        "joining_group": "Du trittst einer bereits bestehenden Gruppe bei!",
+        "login": "Anmelden",
+        "register": "Registrieren",
+        "remember_me": "Angemeldet bleiben",
+        "set_email": "Was ist deine E-Mail?",
+        "set_name": "Wie heißt du?",
+        "set_password": "Setze dein Passwort",
+        "tagline": "Verfolgen, Organisieren und Verwalten deiner Sachen."
+    },
+    "items": {
+        "add": "Hinzufügen",
+        "created_at": "Erstellt am",
+        "custom_fields": "Benutzerdefinierte Felder",
+        "field_selector": "Feldauswahl",
+        "field_value": "Feldwert",
+        "first": "Erste",
+        "include_archive": "Archivierte Elemente einbeziehen",
+        "last": "Letzte",
+        "negate_labels": "Ausgewählte Etiketten negieren",
+        "next_page": "Nächste Seite",
+        "no_results": "Keine Elemente gefunden",
+        "options": "Optionen",
+        "order_by": "Sortieren nach",
+        "pages": "Seite { page } von { totalPages }",
+        "prev_page": "Vorherige Seite",
+        "query_id": "Abfrage Asset-ID-Nummer: { id }",
+        "reset_search": "Suche zurücksetzen",
+        "results": "{ total } Ergebnisse",
+        "tip_1": "Standort- und Etikettenfilter verwenden die 'ODER'-Operation. Wenn mehr als eines ausgewählt ist, wird\n nur eines für eine Übereinstimmung benötigt.",
+        "tip_2": "Suchen, die mit '#' beginnen, fragen nach einer Asset-ID (Beispiel '#000-001')",
+        "tip_3": "Feldfilter verwenden die 'ODER'-Operation. Wenn mehr als eines ausgewählt ist, wird nur eines\n für eine Übereinstimmung benötigt.",
+        "tips": "Tipps",
+        "tips_sub": "Suchtipps",
+        "updated_at": "Aktualisiert am"
+    },
     "profile": {
+        "active": "Aktiv",
         "change_password": "Passwort ändern",
+        "currency_format": "Währungsformat",
         "current_password": "Aktuelles Passwort",
-        "new_password": "Neues Passwort",
-        "notifiers": "Melder",
-        "notifiers_sub": "Benachrichtigungen über anstehende Wartungserinnerungen erhalten",
-        "url": "URL",
+        "delete_account": "Konto löschen",
+        "delete_account_sub": "Löschen Sie Ihr Konto und alle zugehörigen Daten. Dies kann nicht rückgängig gemacht werden.",
         "enabled": "Aktiviert",
-        "test": "Test",
         "gen_invite": "Einladungslink generieren",
-        "user_profile": "Benutzerprofil",
-        "user_profile_sub": "Benutzer einladen und Ihr Konto verwalten.",
-        "active": "Aktiv",
+        "group_settings": "Gruppeneinstellungen",
+        "group_settings_sub": "Geteilte Gruppeneinstellungen. Möglicherweise müssen Sie Ihren Browser aktualisieren, damit einige Einstellungen wirksam werden.",
         "inactive": "Inaktiv",
+        "new_password": "Neues Passwort",
         "notifier_modal": "{ type, select, true {Edit} false {Create} other {Other}} Melder",
```

Comment:
Edit, Create and Other should be updated with the language specific words to replace them.